components/org.apache.stratos.autoscaler/src/main/java/org/apache/stratos/autoscaler/applications/ApplicationUtils.java

```diff
@@ -259,7 +200,7 @@ public static PayloadData createPayload(String appId, String groupName, Cartridg
         return payloadData;
     }
 
-    private static BasicPayloadData createBasicPayload(String appId, String groupName, CartridgeInfo cartridge,
+    private static BasicPayloadData createBasicPayload(String appId, String groupName, Cartridge cartridge,
```

Comment:
The same comment I'd made previously upon introducing a bean class to wrap all these parameters (if applicable) and pass it into the method as an argument would apply here as well.